File: hockeyclub/src/main/java/Stick.java
public class Stick extends Good {
    private double length;

    public Stick(final GoodType goodType, final GoodName name, final int price, final String material,
                 final double weight, final String manufucturer,
                 final String color, final int amount, final double length) {

        super(goodType, name, price, material, weight, manufucturer, color, amount);
        setLength(length);
    }

    public String getHeaders() {
        return super.getHeaders() + "," + "length";
    }

    public String toCVS() {
        return super.toCVS() + "," + getLength();
    }

    public final double getLength() {
        return length;
    }

    public final void setLength(final double length) {
        this.length = length;
    }

}

File: Python lab2/goods/Stick.py
from goods.Good import *
from enums.GoodType import *
from enums.GoodName import *


class Stick(Good):
    goods_type = GoodType.FORANY
    goods_name = GoodName.STICK

    def __init__(self, length, name, price, material, weight, manufacturer, color, amount):
        self.length = length
        Good.__init__(self, name, price, material, weight, manufacturer, color, amount)

    def __str__(self):
        return "Name: " + str(self.name) + " price: " + str(self.price) + " length:" + str(self.length)